### PR TITLE
Theme: add adwaita-light theme

### PR DESCRIPTION
**Pull request by AbrA-K**
Fixes my Issue #10631.
The colors I used are from the adwaita-dark theme, not from the gtksourceview5 Adwaita theme, because I felt the colors had a better contrast to the background that way. These and other changes where mentioned in my issue (first post) - if you want a more direct analog to the dark theme, I could change this to be more in line with the original.

Lastly, I removed the line with the author of the dark theme. They didn't make adwaita-light, but It's HEAVILY based on that.

### CODE REVIEW

**Comment by brielov** [comment]
Can we get this merged?

**Review by the-mikedavis — COMMENTED** [review]
Can this be done with theme inheritance instead? See `gruvbox_light` as an example: you can use `inherits = "adwaita-dark"` and then overwrite the colors in the palette and any styles that don't match `adwaita-dark`

**Comment by brielov** [comment]
@the-mikedavis I think @AbrA-K ended up not sharing colors with `adwaita-dark` but rather using other colors from `gtksourceview` which would make inheritance rather useless in this case, but **I might be mistaken**.

**Comment by AbrA-K** [comment]
No I've copy&pasted the colors from `adwaita-dark`.

I'll change it to use inheritance once I have the time - which should be either later today or tomorrow.

**Comment by AbrA-K** [comment]
I'm using inheritance now + I saw that it was hard to see when you are in insert mode (the cursor was black in normal&insert mode) so I changed that to a yellow when in insert mode. I hope I didn't miss anything else.

File: runtime/themes/adwaita-light.toml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+inherits="adwaita-dark"
+"attribute" = "orange_6"
```
Comment (the-mikedavis) [inline runtime/themes/adwaita-light.toml:2]: It looks like this isn't defined in `adwaita-dark`'s palette or the palette below

Comment (AbrA-K) [reply runtime/themes/adwaita-light.toml]: that's a bit embarrassing haha!

it's fixed now - tried it on a html file and it looks better/more colorfull

**Review by the-mikedavis — APPROVED** [review]

**Review by ErinvanderVeen — APPROVED** [review]

**Pull request merged**